unix/runner

```diff
@@ -107,5 +137,20 @@ try:
     TQM.run(PLAY)
     print(json.dumps(TOTAL_RESULTS, indent=4))
 finally:
+
+    results_callback = RESULTS_CALLBACK
+    eprint("UP ***********")
+    for host, result in results_callback.host_ok.items():
+        eprint('{0}'.format(host))
+
+    eprint("FAILED *******")
+    for host, result in results_callback.host_failed.items():
+        eprint('{0} >>> {1}'.format(host, result._result['msg']))
+        # print('{0} >>> {1}'.format(host, result._result)) # use this to deep dive errors
```

Comment:
Referencing to this in the docs as an option to enable for troubleshooting would be helpful. 

Maybe this would be in a separate readme or the existing one. But sharing it outside of the code might help. Also fine I think if it is in another PR.

Reply:
@prees1 https://github.com/tidalmigrations/machine_stats/pull/11/commits/4e69522cf7d043ec4ae22870642fe42e00ee5978#diff-227db1e259b565fac16490348caf8935R155

Reply:
oh well thats simple :)